OpenSim/Actuators/MuscleFirstOrderActivationDynamicModel.cpp

```diff
@@ -49,24 +48,9 @@ MuscleFirstOrderActivationDynamicModel(double tauActivation,
     std::string name = muscleName + "_activation";
     setName(name);
 
-    SimTK_ERRCHK1_ALWAYS(tauActivation>SimTK::SignificantReal
-                         && tauDeactivation>SimTK::SignificantReal,
-        "MuscleFirstOrderActivationDynamicModel::"
-        "MuscleFirstOrderActivationDynamicModel", "%s: Activation and "
-        "deactivation time constants must be greater than 0",
-        name.c_str());
-
-    SimTK_ERRCHK1_ALWAYS(minActivation >= 0
-                         && minActivation < 1.0-SimTK::SignificantReal,
```

Comment:
You could also use [SimTK_VALUECHECK](http://chrisdembia.github.io/simbody/ExceptionMacros_8h.html#afa5f829126c13e6ea9fa06701c9ff992).